templates/fluent-resource/en-US/security.ftl

```diff
@@ -0,0 +1,112 @@
+### Translation file for page: https://www.rust-lang.org/policies/security
+### templates/policies/security.hbs
+
+security-page-heading = Security policy
+
+security-reporting-heading = Reporting
+security-reporting-link = email security@rust-lang.org
+security-reporting-p1 =
+        Safety is one of the core principles of Rust, and to that end, we would
+        like to ensure that Rust has a secure implementation. Thank you for
+        taking the time to responsibly disclose any issues you find.
+security-reporting-p2 =
+        All security bugs in the Rust distribution should be reported by email
+        to <a href="mailto:security@rust-lang.org">security@rust-lang.org</a>.
+        This list is delivered to a small security team. Your email will be
+        acknowledged within 24 hours, and you’ll receive a more detailed
+        response to your email within 48 hours indicating the next steps in
+        handling your report. If you would like, you can encrypt your report
+        using <a href="/static/keys/rust-security-team-key.gpg.ascii">our public
+        key</a>. This key is also
+        <a href="https://pgp.mit.edu/pks/lookup?op=vindex&amp;search=0xEFB9860AE7520DAC">On
+        MIT’s keyserver</a> and <a href="#key">reproduced below</a>.
+security-reporting-p3 =
+        This email address receives a large amount of spam, so be sure to use a
+        descriptive subject line to avoid having your report be missed. After
+        the initial reply to your report, the security team will endeavor to
+        keep you informed of the progress being made towards a fix and full
+        announcement. As recommended by
+        <a href="https://en.wikipedia.org/wiki/RFPolicy">RFPolicy</a>, these
+        updates will be sent at least every five days. In reality, this is more
+        likely to be every 24-48 hours.
+security-reporting-p4 =
+        If you have not received a reply to your email within 48 hours, or have
+        not heard from the security team for the past five days, there are a few
+        steps you can take (in order):
+security-reporting-li1 =
+        Contact the current security coordinator
+        ({ -security-coordinator-email-anchor }
+        ({ -security-coordinator-public-key-anchor })) directly.
+security-reporting-li2 =
+        Contact the back-up contact ({ -backup-security-contact-email-anchor }
+        ({ -backup-security-contact-public-key-anchor })) directly.
+security-reporting-li3 =
+        Post on the <a href="https://internals.rust-lang.org/">internals forums</a>
+security-reporting-p5 =
+        Please note that the discussion forums are public areas. When escalating
+        in these venues, please do not discuss your issue. Simply say that
+        you’re trying to get a hold of someone from the security team.
+
+## Security coordinator email addreses and links to their public keys
+security-public-key = public key
+-security-coordinator-email-anchor =
```

Comment:
These kinds of links could potentially be pulled out into [`{ $fluent-variables }`](https://projectfluent.org/fluent/guide/variables.html) in the future?

Reply:
Yeah, please move these all into `core.ftl`.

I'll rejigger this and move them outside of the language folders later.